frontend/package.json

```diff
@@ -14,24 +14,21 @@
     "build": "ember build",
     "lint:hbs": "ember-template-lint .",
     "lint:js": "eslint .",
-    "start": "ember serve --proxy https://app.wikonnect.org/",
-    "start:local": "ember serve  --proxy http://localhost:3000",
+    "start": "ember serve  --proxy http://localhost:3000",
```

Comment:
I am cool with this if it will work on production

Reply:
It will work on prod, I just dont like proxies to the server